DevoxxClientMobile/src/main/resources/com/devoxx/common.css

```diff
@@ -147,37 +150,89 @@
-    /*-fx-background-color: -track-cloud-server, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-devices {*/
-    /*-fx-background-color: -track-devices, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-client {*/
-    /*-fx-background-color: -track-client, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-devtools {*/
-    /*-fx-background-color: -track-devtools, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-devops-methodologies {*/
-    /*-fx-background-color: -track-devops-methodologies, white;*/
-/*}*/
-/*.list-cell.schedule-cell:track-unknown {*/
-    /*-fx-background-color: -track-unknown, white;*/
-/*}*/
+
+.list-cell.schedule-cell:track-color0 {
+    -fx-background-color: -track-color0, white;
+}
+.list-cell.schedule-cell:track-color0:color {
+    -fx-background-color: -track-color0, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color1 {
+    -fx-background-color: -track-color1, white;
+}
+.list-cell.schedule-cell:track-color1:color {
+    -fx-background-color: -track-color1, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color2 {
+     -fx-background-color: -track-color2, white;
+}
+.list-cell.schedule-cell:track-color2:color {
+    -fx-background-color: -track-color2, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color3 {
+     -fx-background-color: -track-color3, white;
+}
+.list-cell.schedule-cell:track-color3:color {
+    -fx-background-color: -track-color3, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color4 {
+     -fx-background-color: -track-color4, white;
+}
+.list-cell.schedule-cell:track-color4:color {
+    -fx-background-color: -track-color4, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color5 {
+     -fx-background-color: -track-color5, white;
+}
+.list-cell.schedule-cell:track-color5:color {
+    -fx-background-color: -track-color5, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color6 {
+     -fx-background-color: -track-color6, white;
+}
+.list-cell.schedule-cell:track-color6:color {
+    -fx-background-color: -track-color6, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color7 {
+     -fx-background-color: -track-color7, white;
+}
+.list-cell.schedule-cell:track-color7:color {
+    -fx-background-color: -track-color7, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color8 {
+     -fx-background-color: -track-color8, white;
+}
+.list-cell.schedule-cell:track-color8:color {
+    -fx-background-color: -track-color8, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell:track-color9 {
+    -fx-background-color: -track-color9, white;
+}
+.list-cell.schedule-cell:track-color9:color {
+    -fx-background-color: -track-color9, derive(-fx-control-inner-background,-7%);
+}
+
+.list-cell.schedule-cell > .list-tile > Group > .label {
```

Comment:
`Group` can be replaced by `.primary-graphic` (just to avoid any possible conflict)